tdrs-frontend/manifest.yml

```diff
@@ -2,10 +2,10 @@
 version: 1
 applications:
 - name: tdp-frontend
+  buildpacks:
```

Comment:
Again we are being explicit with which build pack we are using so we don't risk possilby confusing the auto detection mechanisim